src/EnergyPlus/SQLiteProcedures.cc

```diff
@@ -2672,9 +2672,9 @@ int SQLiteProcedures::sqlitePrepareStatement(sqlite3_stmt *&stmt, const std::str
     return rc;
 }
 
-int SQLiteProcedures::sqliteBindText(sqlite3_stmt *stmt, const int stmtInsertLocationIndex, const std::string &textBuffer)
+int SQLiteProcedures::sqliteBindText(sqlite3_stmt *stmt, const int stmtInsertLocationIndex, const std::string_view &textBuffer)
 {
-    int rc = sqlite3_bind_text(stmt, stmtInsertLocationIndex, textBuffer.c_str(), -1, SQLITE_TRANSIENT);
+    int rc = sqlite3_bind_text(stmt, stmtInsertLocationIndex, format(textBuffer).c_str(), -1, SQLITE_TRANSIENT);
```

Comment:
Since it passes a string into a C library, I ended up using a format statement at the deepest level.

Reply:
`std::string_view` supports the `.data()` method which I is functionally equivalent to `.c_str()` for this purpose (you may need to add `static_cast<const char *>`). Can you try that and see if it works? Thx!

Reply:
Since the `data()` method has some caveats about passing it down to C libraries, I had advised her it would be OK to create a string, or a const char* or whatever, to pass down into the C library.  As long as the underlying data is a null terminated string (it should be, right??) then passing `data()` is fine.  However this resolves it will be good to set the precedent for if we have to do this in other places.  Thanks for watching this @amirroth!

Reply:
I see. It's not that hard to create a run-on string in C (a character buffer that is shorter than the string you want will often do it, actually it will often do worse than that). It's harder to do in C++ with `std::string` and `std::string_view` so I would assume this doesn't happen in EnergyPlus, but who knows. Something to put in the parking lot and look at later, maybe. Thx.